english/udacity/udacity.md

```diff
@@ -1 +1,3 @@
-Alas, there is not, you're stuck. Get Caroline on it!
\ No newline at end of file
+Alas, there is not, you're stuck. Get Caroline on it!
+
+Click [here](https://github.com/cbuckey-uda) to visit Caroline's github page!
```

Comment:
It's hard for Casey to tell whether people are linking to their own forks of the create-your-own-adventure repo, which I wouldn't want. But this link is fine.